lib/nerdtree/menu_controller.vim

```diff
@@ -71,9 +71,9 @@ endfunction
 "change the selection (if appropriate) and return 1 if the user has made
 "their choice, 0 otherwise
 function! s:MenuController._handleKeypress(key)
-    if a:key == 'j'
+    if a:key == 'j' || a:key == g:NERDTreeMenuDown
```

Comment:
Change this to
```vim
if a:key == g:NERDTreeMenuDown
```
and initialize the variable in this section: https://github.com/scrooloose/nerdtree/blob/master/plugin/NERD_tree.vim#L116-L150